patches/README.md

```diff
@@ -0,0 +1,61 @@
+# MkDocs Theme Patches
+
+This directory contains patches for the files contained in the MkDocs theme
+[overrides](../overrides) directory. This enables upgrading to new versions of
+the theme in the future without manually trying to resolve the differences
+between the originals and our custom files.
+
+The patches are currently based on v5.5.6 of mkdocs-material. Originals can be
+found in your Python environment's `Lib/site-packages/material` directory.
+
+## Modifications
```

Comment:
Does anything need to go here? If not maybe we can put some empty text or remove the section all-together.

Reply:
It gives it some hierarchy by grouping the individual files under a level 2 heading instead of putting them at the same level as the later sections. It would probably be odd to put a level 3 heading directly under a level 1.